### DISCUSSION TITLE
Problems to configure rating component with FormControl in Angular 17

### DISCUSSION THREAD

**Discussion by ottopic** [saadeghi/daisyui#2983] [Q&A] [answered]
When "editing" the rating, everything works correctly, i.e., the correct stars are illuminated based on ratingValue > 0.
However, when I am in the process of adding new rating, so I would like all the stars off with ratingValue = 0 they are all lit up and I cannot find a way to make it start from 0.  This problem is caused using formControlName="rating" because is part of FormGroup.
I have tried using [value]="0" or [value]="ratingValue" but nothing change.

Any ideas to help me?

![image](https://github.com/saadeghi/daisyui/assets/5537554/2b70b198-fa86-4d4c-86c6-b263d0eae595)

**Comment by saadeghi** [comment, marked as answer]
Use different `name` attributes for the new group. They are radio inputs so just like radio inputs, you can group them by `name`

Let me know if you have a question.
